pygmt/datasets/tile_map.py

```diff
@@ -79,6 +90,14 @@ def load_tile_map(region, zoom="auto", source=None, lonlat=True, wait=0, max_ret
         will stop trying to fetch more tiles from a rate-limited API [Default
         is ``2``].
 
+    zoom_adjust : int or None
+        Optional. The amount to adjust a chosen zoom level if it is chosen
+        automatically. Values outside of -1 to 1 are not recommended as they can lead to
+        slow execution. [Default is ``None``].
```

Comment:
After PR #2813, the parameter type and default value can be automatically determined from type hints:
```suggestion
    zoom_adjust
        The amount to adjust a chosen zoom level if it is chosen
        automatically. Values outside of -1 to 1 are not recommended as they can lead to
        slow execution.
```

Reply:
Ok, done at 9170ea08b8c5b532f20533005d05e3cc20bec993 and 90a4d4ffc6e8882592acba73d38f9f342bbe308d for both `tilemap` and `load_tile_map`.